apps/sms/services/test/unit/moz_mobile_message/moz_mobile_message_shim_test.js

```diff
@@ -164,17 +131,39 @@ suite('MozMobileMessageShim >', function() {
       );
     });
 
-    test('retrieveMMS', function() {
-      var args = [1];
+    suite('retrieveMMS', function() {
+      var args;
 
-      assert.equal(
-        MozMobileMessageShim.retrieveMMS(...args),
-        MockNavigatormozMobileMessage.retrieveMMS.firstCall.returnValue
-      );
-      sinon.assert.calledWith(
-        MockNavigatormozMobileMessage.retrieveMMS,
-        ...args
-      );
+      setup(function() {
+        args = [1];
+      });
+
+      test('success', function() {
+        MockNavigatormozMobileMessage.retrieveMMS.returns(Promise.resolve({
+          id: 'fake message'
+        }));
```

Comment:
__nit:__ also should not we just use ```var messageId = 'fake_message_id'``` instead of these ```...args```? Looks a bit over-engineered and confusing (and then we don't need ```setup``` at all).